src/components/NcRadioGroup/NcRadioGroup.vue

```diff
@@ -43,6 +49,12 @@ provide(INSIDE_RADIO_GROUP_KEY, computed(() => ({
 	onUpdate,
 })))
 
+watchEffect(() => {
+	if (!props.label) {
+		warn('[NcRadioGroup] For accessibility reasons a proper "label" has to be set.')
+	}
+})
+
```

Comment:
This `watch` duplicates Vue's built-in required prop check...

Reply:
Can it be done just in setup() or a mounted() hook?

Reply:
> Can it be done just in setup() or a mounted() hook? should be enough for a dev to catch it

Sure but there should be no overhead because if the label does not change the watch is not executed.
If the label changes the `onMounted` hook would not catch it.
We can also remove this completely because it would only trigger if developers "hack" this explicitly.